### PR TITLE
For intalling ambari server

### PR DESCRIPTION
**Pull request by taffazzel**
@CloudwickShared/cloudwick Please review

### CODE REVIEW

**Comment by StayerX** [comment]
Looks good. Make it an executable script (#!/bin/bash)

**Comment by StayerX** [comment]
This is for #5

**Comment by bhavintandel** [comment]
pls change apt-get to wget

**Comment by StayerX** [comment]
@bhavintandel very good observation!
@taffazzel I agree with @bhavintandel. Also add this line `rm /etc/yum.repos.d/ambari.repo`

The final file should look like this:

```
#!/bin/bash
cd /etc/yum.repos.d/
rm /etc/yum.repos.d/ambari.repo
wget http://public-repo-1.hortonworks.com/ambari/centos6/2.x/updates/2.2.1.0/ambari.repo
yum install ambari-server
ambari-server setup -j ambari-server setup -j /usr/java/latest/
```

File: ambari-server.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cd /etc/yum.repos.d/
+apt-get http://public-repo-1.hortonworks.com/ambari/centos6/2.x/updates/2.2.1.0/ambari.repo
```
Comment (StayerX) [inline ambari-server.sh:2]: should not be `apt-get`, should be `wget`. Please test the code on your server.

**Comment by StayerX** [comment]
@taffazzel  Please stop merging things that should not be merged.

**Comment by despoinak1** [comment]
@taffazzel  has merged the file but the changes that need to be made are not yet done.

**Pull request merged**